<launch>
    <arg name="map_file" default="$(find daart)/resources/maps/ensta2.yaml" />

    <include file="$(find stdr_robot)/launch/robot_manager.launch" />
    <include file="$(find stdr_gui)/launch/stdr_gui.launch"/>

    <node type="stdr_server_node" pkg="stdr_server" name="stdr_server" output="screen" args="$(arg map_file)"/>

    <node pkg="stdr_robot" type="robot_handler" name="robot0" args="add $(find daart)/resources/robots/omni_robot.xml 7 5 0" />
    <!--<node pkg="stdr_robot" type="robot_handler" name="robot0" args="add $(find intervalar_localization)/resources/robots/omni_robot.xml 6.5 8.5 -1.57" />-->
    <node name="rviz" type="rviz" pkg="rviz" args="-d $(find daart)/config/config.rviz" />

    <group ns="robot0">
        <!-- <include file="$(find intervalar_localization)/launchers/joy_teleop.launch"/> -->
        <!-- <include file="$(find intervalar_localization)/launchers/amcl.launch"/> -->
        <remap from="/robot0/map" to="/map" />
        <remap from="/robot0/move_base_simple/goal" to="/move_base_simple/goal" />
        <remap from="/robot0/laser_0" to="/scan" />
        <!--<remap from="/robot0/odom" to="/odom" />-->
        <!--
        <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <remap from="cmd_vel" to="/robot0/cmd_vel" />
        <rosparam file="$(find intervalar_localization)/config/common_cost_map_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find intervalar_localization)/config/common_cost_map_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find intervalar_localization)/config/local_cost_map.yaml" command="load" />
        <rosparam file="$(find intervalar_localization)/config/global_cost_map.yaml" command="load" />
        <rosparam file="$(find intervalar_localization)/config/base_local_planner_params.yaml" command="load" />
        </node>-->
    </group>

    <!-- <node pkg="tf" type="static_transform_publisher" name="odom2map"  args="7 13 0 0 0 0  map       odom          1000" />
    <node pkg="tf" type="static_transform_publisher" name="laser2map" args="0 0 0 0 0 0   base_link laser 				 1000" />
    <node pkg="tf" type="static_transform_publisher" name="imu2map"   args="0 0 0 0 0 0   base_link base_imu_link 1000" /> -->
</launch>
